app/assets/stylesheets/components/navbar.scss

```diff
@@ -169,6 +169,11 @@
   }
 }
 
+@media (min-width: 768px) and (max-width: 1024px)  {
+      .navbar{
```

Comment:
The indentation seems to be a bit off, can this and its body be indented by 2 spaces just so its consistent with the rest of the file please @DoviMaj

Reply:
I've done it. Is it good now?